opslevel/resource_opslevel_team.go

```diff
@@ -108,70 +112,87 @@ func reconcileTeamAliases(d *schema.ResourceData, team *opslevel.Team, client *o
 	return nil
 }
 
-func collectMembersFromTeam(team *opslevel.Team) []string {
-	members := []string{}
+func collectMembersFromTeam(team *opslevel.Team) []opslevel.TeamMembershipUserInput {
+	members := []opslevel.TeamMembershipUserInput{}
 
-	for _, user := range team.Members.Nodes {
-		members = append(members, user.Email)
+	for _, user := range team.Memberships.Nodes {
+		member := opslevel.TeamMembershipUserInput{
+			User: opslevel.UserIdentifierInput{
+				Email: user.User.Email,
+			},
+			Role: string(user.Role),
```

Comment:
I'm not sure this is the same "role" - we should check with catalog creation - but `user.Role` is their "permissions" and `membership.Role` is something entirely different - manager, or tech_lead, or IC, etc